src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -28,6 +27,7 @@ public abstract class FormulaType
 
         public static FormulaType Number { get; } = new NumberType();
 
+        [SuppressMessage("Naming", "CA1720:Identifiers should not contain type names", Justification="n/a")]
```

Comment:
We may want to disable CA1720 generally, this product is a compiler, and it's reasonable for some identifiers to be type names

Reply:
Agreed, removed